### DISCUSSION TITLE
스터디 생성시 입력값에 대한 유효성 검사

### DISCUSSION THREAD

**Discussion by airman5573** [woowacourse-teams/2022-moamoa#142] [공유]
- 1자 <= 스터디 타이틀("title") <= 20자
- 1자 <= 스터디 소개글("description") <= 20000자
- 1자 <= 스터디 한줄 소개("excerpt") <= 50자
- 1명 <= 스터디 최대 인원("maxMemberCount") <= 100명

이상입니다 :D

**Comment by jaejae-yoo** [comment]
:D

**Comment by tco0427** [comment]
20자로 하니 생각보다 너무 짧을 수도 있겠다는 생각을 해요..!! 기존의 2022-looking-for-the-sound-of-an-object 는 어림도 없고.. looking-for-sound-of-objects 때에도 20자를 초과하네요.. (Sample 데이터 이기는 하지만)

사용자의 자유도를 조금 더 높이기 위해 30자는 어떨까요??
